<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppDetail</class>
 <widget class="QMainWindow" name="AppDetail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FlashLauncher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#centralWidget {
background-image: url(:/background/background-thin.png);
}</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="styleSheet">
       <string notr="true">#widget {
background-image: url(:/background/gameinfobox.png);
background-repeat: none;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="imgpic">
           <property name="styleSheet">
            <string notr="true">#imgpic {
margin-top: 16px;
margin-left: 10px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="icons.qrc">:/icons/missingicon92.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="desc">
           <property name="styleSheet">
            <string notr="true">#desc {
	margin-top: 5px
}</string>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="webB">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#webB {
     border-width: 0px;
  	 alternate-background-color: rgba(255, 255, 255, 0);
 	 background-color: rgba(255, 255, 255, 0);
     border-radius: 0px;
     padding: 0px;

}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/background/dummypix.png</normaloff>:/background/dummypix.png</iconset>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCommandLinkButton" name="okB">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#okB {
     border-width: 0px;
  	 alternate-background-color: rgba(255, 255, 255, 0);
 	 background-color: rgba(255, 255, 255, 0);
     border-radius: 0px;
     padding: 0px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/background/dummypix.png</normaloff>:/background/dummypix.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="instructions">
         <property name="styleSheet">
          <string notr="true">#instructions {
	background-color: rgba(255, 255, 255, 0);
	margin-left: 50px;
	margin-top: 10px;
	margin-right: 50px;
    color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'DejaVu Sans'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Instructions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="statusbar" native="true">
         <property name="styleSheet">
          <string notr="true">#statusbar {
    background-image: url(none);
	background-color: rgb(16, 16, 16);
}</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="engine">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>QtWebKit</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>QtWebKit (html loader)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MicroB</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>MicroB (html loader)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>KMPlayer</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="qualityCB">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>High</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Medium</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Low</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>563</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionSend_config_via_email"/>
    <addaction name="actionCopy_config_to_clipboard"/>
    <addaction name="actionDelete"/>
    <addaction name="actionHide"/>
    <addaction name="actionEdit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <action name="actionSend_config_via_email">
   <property name="text">
    <string>Send config via email</string>
   </property>
  </action>
  <action name="actionCopy_config_to_clipboard">
   <property name="text">
    <string>Copy config to clipboard</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
